cypress/integration/pages/mostReadPage/tests.js

```diff
@@ -0,0 +1,23 @@
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing features that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
+  describe(`No testsThatFollowSmokeTestConfig to run for ${service} ${pageType}`, () => {
+    describe(`Tests for ${service} ${pageType}`, () => {
```

Comment:
Should we make sure both of these are covered by integration tests, and then just run the shared tests for this page type at the moment?

Reply:
yep, I agree. Tests are already covered here: `src/integration/pages/mostReadPage/crossPlatformTests.js`